.github/actions/bench/action.yml

```diff
@@ -61,8 +61,21 @@ runs:
     - name: Run benchmark
       shell: ${{ env.SHELL }}
       run: |
-        tests bench -c ${{ inputs.perf }} --cross-prefix="${{ inputs.cross_prefix }}" --cflags="${{ inputs.cflags }}" --arch-flags="${{ inputs.archflags }}" --opt="${{ inputs.opt }}" -v --output=output.json ${{ inputs.bench_extra_args }}
-        tests bench --components -c ${{ inputs.perf }} --cross-prefix="${{ inputs.cross_prefix }}" --cflags="${{ inputs.cflags }}" --arch-flags="${{ inputs.archflags }}" --opt="${{ inputs.opt }}" -v --output=output.json ${{ inputs.bench_extra_args }}
+        catch() {
+          local line_no="$1"
+          echo "::error title=Failed at $line_no: $?"
+        }
+        trap 'catch $LINENO' ERR
```

Comment:
Sorry, what is this doing?

Reply:
Ah, I thought I can make the error annotation better, but turns out this didn't work out as I expected, I'll remove it for now